### PR TITLE
Minor changes to G33

### PR DESCRIPTION
**Pull request by LVD-AC**
- update comment section
- redefined P5 grid
- minor clean to probe radius routine
- updated EEPROM version to V37 according remark in #6517

### CODE REVIEW

File: Marlin/Marlin_main.cpp [hunk 1]
```diff
@@ -5198,28 +5198,26 @@ inline void gcode_G28() {
...
-            start_circles += (pp_greather_5 ? (zig_zag ? 0.5 : -0.5) : 0);
-            end_circles = -start_circles;
             zig_zag = !zig_zag;
-            z_at_pt[axis] /= (pp_equals_7 ? (zig_zag ? 4.0 : 3.0) :
-                              pp_equals_6 ? (zig_zag ? 3.0 : 2.0) : pp_equals_5 ? 3 : 1);
+            z_at_pt[axis] /= (2 * offset_circles + 1);
           }
         }
-        if (pp_greather_3 && !pp_equals_5) // average intermediates to tower and opposites
+        if (pp_greather_3) // average intermediates to tower and opposites
```
Comment (thinkyhead) [inline Marlin/Marlin_main.cpp:5220]: "greater"
Who vetted this? 😉

**Comment by thinkyhead** [comment]
I'll patch this up to fix spelling.

The `pp_equals` and `pp_greater` items should really be named meaningfully — according to what they do. I'll try to determine that from the code comments.

**Comment by thinkyhead** [comment]
#### Documentation style note (for developers generally):

When writing commentary, pretend each statement begins with "This will…" (rather than just "This…") and begin there. So, for example, instead of a comment that says:

…This…
```cpp
// Opens the file and sends it to serial
```
Use instead:

…This will…
```cpp
// Open the file and send it to serial
```

**Comment by LVD-AC** [comment]
pp_ stands for the "probe_points" variable which holds the Pn value; I do not have a Shakespeare spellcheck in C++ unfortunally; I'll guest you are the native English speaker here, pardon my French. ;) I did not put the testing with probe_mode (which can be negative) into bool constants since there's only a couple of those in the whole code and as such there is a clear difference between the two.

**Comment by thinkyhead** [comment]
> I do not have a Shakespeare spellcheck in C++ unfortunately

I'd rather you didn't keep doing that. It implies that I'm being unreasonable, which I'm not. I'm just trying to be thorough.

**Pull request closed without merging**